ui-frontend/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -566,12 +574,7 @@ module.exports = Backbone.AssociatedModel.extend({
     const lat = this.get('lat'),
       lon = this.get('lon')
 
-    if (
-      (!Drawing.isDrawing() && this.get('locationType') !== 'latlon') ||
-      !this.isLatLonValid(lat, lon)
-    ) {
-      return
-    }
+    if (!this.isLatLonValid(lat, lon)) return
```

Comment:
This allows usng and utmUps to be updated for point radii outside of drawing

Reply:
The conditional removed was initially added in this PR: https://github.com/codice/ddf/pull/1875 

(https://github.com/glenhein/ddf/blob/86dceb5ec0515ad4a4389d06e93ab1a5b1fba08c/catalog/ui/catalog-ui-search/src/main/webapp/component/location-old/location-old.js#L310)

There are some concerns related to the point radius in the PR, including https://github.com/codice/ddf/pull/1875#pullrequestreview-34350427. I'm not sure if that conditional is needed or not in preventing a regression somewhere. We might want to follow the test instructions that Glen wrote for point radius testing in the PR above to make sure we didn't regress.

Reply:
Good call. I was having trouble tracing this back since the git blame just mentions the move to ddf-ui.

Reply:
Looked into this a bit.
1. Looks like the issue I was seeing was related to my changes. The initialization doesn't work without removing these lines, because it prevents the usng and utm fields from being updated.
2. I'm not seeing the issues with drawing that Will ran into, but editing utm is interrupted because this method gets past this point. So, if you start typing coordinates, this stops you and updates your input immediately.

I should be able to just factor out the code I need from this for updating the other coordinate systems and just use it in the init method though.